src/CommonITILObject.php

```diff
@@ -7358,6 +7359,8 @@ public function getTimelineItemtypes(): array
 
         $itemtypes = [];
 
+        $use_mentions = UserMention::getRestrictedUsers($this);
```

Comment:
I had to read the code multiple times to figure out what this variable contains. I guess the naming can be improved here.

`use_mentions` => `mentions_options`
`UserMention::getRestrictedUsers()` => `UserMention::getMentionOptions()`